### PR TITLE
[Blazor wasm]`logout` if token expired.

### PR DESCRIPTION
**Pull request by maliming**
https://support.abp.io/QA/Questions/5071

https://github.com/abpframework/abp/assets/6908465/f97406d6-264e-4162-b09a-d1d52f153402

### CODE REVIEW

File: framework/src/Volo.Abp.AspNetCore.Components.WebAssembly/Volo/Abp/AspNetCore/Components/WebAssembly/AbpAspNetCoreComponentsWebAssemblyModule.cs [hunk 1]
```diff
@@ -18,7 +19,8 @@ namespace Volo.Abp.AspNetCore.Components.WebAssembly;
 [DependsOn(
     typeof(AbpAspNetCoreMvcClientCommonModule),
     typeof(AbpUiModule),
-    typeof(AbpAspNetCoreComponentsWebModule)
+    typeof(AbpAspNetCoreComponentsWebModule),
+    typeof(AbpEventBusModule)
```
Comment (hikalkan) [inline framework/src/Volo.Abp.AspNetCore.Components.WebAssembly/Volo/Abp/AspNetCore/Components/WebAssembly/AbpAspNetCoreComponentsWebAssemblyModule.cs]: Why adding this to dependencies? Volo.Abp.AspNetCore.Components.WebAssembly.csproj has no such a project reference.

**Review by hikalkan — CHANGES_REQUESTED** [review]

File: framework/src/Volo.Abp.AspNetCore.Components.WebAssembly/Volo/Abp/AspNetCore/Components/WebAssembly/AbpAspNetCoreComponentsWebAssemblyModule.cs (hunk 1 above)
Comment (maliming) [reply framework/src/Volo.Abp.AspNetCore.Components.WebAssembly/Volo/Abp/AspNetCore/Components/WebAssembly/AbpAspNetCoreComponentsWebAssemblyModule.cs]: I add a `ClientProxyExceptionEventHandler` in this project.

Comment (maliming) [reply framework/src/Volo.Abp.AspNetCore.Components.WebAssembly/Volo/Abp/AspNetCore/Components/WebAssembly/AbpAspNetCoreComponentsWebAssemblyModule.cs]: I will also add the project reference.

Comment (hikalkan) [reply framework/src/Volo.Abp.AspNetCore.Components.WebAssembly/Volo/Abp/AspNetCore/Components/WebAssembly/AbpAspNetCoreComponentsWebAssemblyModule.cs]: If there is no project reference, but you can use the module (that defines ClientProxyExceptionEventHandler), it means you indirectly reference the project. In this case, you should have an indirect module dependency. So, you should not define the module dependency, it should also come indirectly.

**Review by hikalkan — APPROVED** [review]

**Pull request merged**